functions/exercise2/script.js

```diff
@@ -7,4 +7,16 @@ function isPalindrome(phrase) {
     }
     var reversedNormalizedPhrase = normalizedPhrase.split("").reverse().join("");
     return reversedNormalizedPhrase === normalizedPhrase;
+}
+
+function showResult() {
+    var phrase = document.getElementById("phrase").value;
+    try {
+        var result = isPalindrome(phrase);
+        if (result === true) { document.getElementById("result").value = `${phrase} is palindrome :)`; }
+        if (result === false) { document.getElementById("result").value = `${phrase} isn't palindrome :(`; }
```

Comment:
Why do you check if result is false here? Have you heard about if else statement?